test/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,8 @@
-const { booleanToWord } = require("../src");
+const { booleanToWord } = require('../src');
 
-describe("booleanToWord", () => {
-  // how do we create specs again???
+describe('booleanToWord', () => {
+	test('check boolean returns correct string', () => {
+		expect(booleanToWord(true)).toBe('Yes');
+		expect(booleanToWord(false)).toBe('No');
+	});
```

Comment:
I would separated out these expect statements into separate tests blocks. It improves readability and, if the function failed, would make it more obvious why it's failing. e.g.
```
test("returns 'Yes' when passed true", () => {
  expect(booleanToWord(true)).toBe("Yes");
});

test("returns 'No' when passed false", () => {
  expect(booleanToWord(false)).toBe("No");
});
```